### PR TITLE
Week5

### PR DESCRIPTION
**Pull request by stephskas**
storing data in mongodb

### CODE REVIEW

File: 03-task-manager/answers/db/connect.js
Comment (akosasante) [inline 03-task-manager/answers/db/connect.js:9]: Just curious, were these the options that were passed into the connect call in the video? Mainly asking because I've mainly been seeing the following in other assignments, so I'm curious why this one differs:

```
-    useNewUrlParser: true,
-    useCreateIndex: true,
-    useFindAndModify: false,
-    useUnifiedTopology: true
```

The `writeConcern` option you've included here is fine but generally not something I'd be too worried about including at this stage. It tells MongoDb how to handle requests to create or edit data in the database if you have multiple copies (replicas) of the database running

File: 03-task-manager/answers/routes/tasks.js
Comment (akosasante) [inline 03-task-manager/answers/routes/tasks.js:16]: These can be removed; the comment is redundant; and the `route('/').get(getAllTasks)` has already been defined on line 12

File: 03-task-manager/answers/quizAnswers.txt
Comment (akosasante) [inline 03-task-manager/answers/quizAnswers.txt:7]: That's right; the acronym is commonly used because these are the four actions that are most commonly needed when building a REST API application with some sort of persistent backend/database.

**Review by akosasante — APPROVED** [review]
Looks good; left a couple of small comments but after that you should be good to merge this PR (after merging the Week 4 PR)

File: 03-task-manager/answers/db/connect.js
Comment (stephskas) [reply 03-task-manager/answers/db/connect.js:9]: Thanks for the review. I was getting an error (MongoWriteConcernError) - I saved the error message and looked it up and found that I could update my MongoDB connection configuration with this code  to prevent the error. Perhaps I had multiple copies of the database running?

**Pull request merged**
